CHANGELOG.md

```diff
@@ -2,6 +2,20 @@
 
 ## Unreleased
 
+### Improvements
+
+- [#538](https://github.com/cosmos/iavl/pull/538) Use cosmos-db instead of tm-db
+- [#537](https://github.com/cosmos/iavl/pull/537) Add a gitpod development environment so it is easy to work with all supported databases
+
+
+**Linting and cleanup**
+
+- [#542](https://github.com/cosmos/iavl/pull/542) Move keyformat and fastnode to specific folders
```

Comment:
this is a breaking change. Should be listed in `### Breaking Changes section`

Reply:
Makes a billion types of sense, thank you

Reply:
keyformat change?

Or do you mean badger and bolt?

Reply:
external API changed - you moved some functions / types to other package.

Reply:
UP, this should go into Breaking Changes section